### DISCUSSION TITLE
第一次使用配置了azure open ai为什么save不上

### DISCUSSION THREAD

**Discussion by tarkky** [nextai-translator/nextai-translator#1712] [Q&A]
tranlator版本0.4.33
系统windows 11 64， 补丁已经打到最新
azure openai 部署。
配置信息都是在azure openai studio的部署页面获得
模型名称gpt-4
key也是在这个页面的key粘贴进来的。
api URL里面填写的是studio页面的target URL里面的/openai/...前面的部分。
API URL path就是/openai/开始后面的部分粘贴进去了。
然后点击保存。保存按钮就一直转圈。
请问是什么问题？
![6HNWbymQFb](https://github.com/user-attachments/assets/bbeb0f76-7139-455d-a361-275dc6e9019d)

**Comment by wlangw** [comment]
同意的问题

**Comment by Weiiiz** [comment]
Did you solve the problem? I had the same problem.

**Reply by tarkky** [reply]
no, not yet.

**Comment by zhanglibin001** [comment]
一模一样的问题，为什么呀

**Comment by wadewej** [comment]
tranlator版本0.4.33 任何内容设置都save不了，一直在打转，我换成0.4.31正常，希望作者尽快修复，另外这个软件用起来很卡

**Comment by wadewej** [comment]
忘了说了，windows 版本
